tests/WorkOS/WebhookTest.php

```diff
@@ -14,26 +14,51 @@ protected function setUp(): void
 
         $this->withApiKey();
         $this->ap = new Webhook();
+
+        $this->payload = '{"id":"wh_01FGCG6SDYCT5XWZT9CDW0XEB8","data":{"id":"conn_01EHWNC0FCBHZ3BJ7EGKYXK0E6","name":"Foo Corp\'s Connection","state":"active","object":"connection","domains":[{"id":"conn_domain_01EHWNFTAFCF3CQAE5A9Q0P1YB","domain":"foo-corp.com","object":"connection_domain"}],"connection_type":"OktaSAML","organization_id":"org_01EHWNCE74X7JSDV0X3SZ3KJNY"},"event":"connection.activated"}';
+        $this->secret = 'secret';
+        $this->tolerance = 180;
+        $this->time = time();
+        $decodedBody = utf8_decode($this->payload);
+        $signedPayload = $this->time . "." . $decodedBody;
+        $this->expectedSignature = hash_hmac("sha256", $signedPayload, $this->secret, false);
+        $this->sigHeader = 't=' . $this->time . ', v1=' . $this->expectedSignature;
     }
 
     public function testConstructEventWebhook()
```

Comment:
Moved test data to setUp() function as all other tests are using the same variables